content/en/docs/getting-started/_index.md

```diff
@@ -94,9 +94,9 @@ sudo systemctl enable microshift --now
 To access the cluster, install the OpenShift client or kubectl.
 
 ```Bash
-curl -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
-sudo tar -C /usr/local/bin -xzvf oc.tar.gz
-sudo install -t /usr/local/bin {kubectl,oc}
+curl -O https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz
+tar -xvzf openshift-client-linux.tar.gz -C /tmp
+sudo install /tmp/{kubectl,oc} /usr/local/bin
```

Comment:
You don't need separate `tar` and `sudo install` commands, you can just run:

```
sudo tar -xvf openshift-client-linux.tar.gz -C /usr/local/bin oc kubectl
```

Note that you don't need `-z`, either, because `tar` figures out the compression format automatically (so this will do the right thing even if they were to start using `.xz` instead, for example).